tests/scsi/008

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
+
+. tests/zbd/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test block data lifetime support"
+QUICK=1
+
+requires() {
+	_have_fio
```

Comment:
I did trial run for this test case and noticed that it fails due to fio direct write failure when fio does not have [the recent commit you made](https://github.com/axboe/fio/commit/0cfea592fedf0011e695a604a6961e9cbc1fe9b6). So, it's the better to check that fio has the commit. One idea is to add "_have_fio_ver 3 37" here, and wait for (or ask for) the next fio tag release v3.37.

Reply:
`_have_fio_ver 3 37` has been added.